index.bs

```diff
@@ -58,8 +58,7 @@ urlPrefix: https://tools.ietf.org/html/draft-ietf-httpbis-header-structure; spec
         text: sh-token; url: #section-3.9
     type: abstract-op
         text: serialize Structured Header; url: #section-4.1
-        text: Structured Header parsing algorithm; url: #section-4.2.7
-urlPrefix: https://html.spec.whatwg.org/
+        text: Structured Header Token parsing algorithm; url: #section-4.2.6 urlPrefix: https://html.spec.whatwg.org/
```

Comment:
1. The `urlPrefix` bit looks like things might have gotten a little mixed up...

2.  My understanding of Structured Headers is that the entry point for parsing ought to be the general parsing algorithm in 4.2 (https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-14#section-4.2), not the specific types.

Reply:
1. Uh, I don't know how that `urlPrefix` got there...
2. OK. Since the input to that is a byte sequence, we should remove the isomorphic decoding in step 1.